### PR TITLE
Support setter(each(into))

### PR DESCRIPTION
**Pull request by TedDriggs**
This change is based on #214, allowing collection setters to use `Into` for extension items. This change **is** backward-compatible; both the name and nested form of `each` are supported.

cc @uint @andy128k

### CODE REVIEW

File: derive_builder/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
-- Compatibility between `setter(strip_option`) and `setter(each = "name")` #213
+- Change `each` syntax from `#[builder(setter(each = "bar"))]` to
```
Comment (TedDriggs) [inline derive_builder/CHANGELOG.md:6]: Need to change this to reflect it's not breaking.

File: derive_builder/tests/setter_extend.rs [hunk 2]
```diff
@@ -7,11 +7,11 @@ use std::collections::HashMap;
 
 #[derive(Debug, PartialEq, Default, Builder, Clone)]
 struct Lorem {
-    #[builder(setter(each = "foo_append"))]
+    #[builder(setter(each(name = "foo_append")))]
```
Comment (TedDriggs) [inline derive_builder/tests/setter_extend.rs]: Revert this change to make sure old syntax works.

File: derive_builder_core/src/options.rs [hunk 3]
```diff
@@ -33,3 +35,49 @@ impl Default for BuilderPattern {
...
+    pub into: bool,
+}
+
+/// Create `Each` from an attribute's `Meta`.
+///
+/// Two formats are supported:
+///
+/// * `each = "..."`, which provides the name of the `each` setter and otherwise uses default values
+/// * `each(name = "...")`, which allows setting additional options on the `each` setter
+impl FromMeta for Each {
```
Comment (TedDriggs) [inline derive_builder_core/src/options.rs]: This is duplicative with `EachLongOrShort` and should be removed.

**Review by andy128k — APPROVED** [review]
